index.html

```diff
@@ -0,0 +1,321 @@
+    <section class="middle-section">
+      <div class="card1">
+        <div class="snapshoot">
+          <img src="./images/screenshot1.png" alt="screenshot" class="responsive"/>
+        </div>
+        <div class="word-title">
+          <div class="w-title">
+            <h2 class="name">Thomas</h2>
+            <ul class="canop">
+              <li>CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div>
+            <ul class="languages">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/see-project.png" alt="button" class="tap">
+        </div>
+      </div>
+      <div class="card2">
+        <div class="snapt">
+          <img src="./images/snapshot4.png" alt="snapshot" class="responsive"/>
+        </div>
+        <div class="titol">
+          <div class="w-title">
+            <h2 class="name">Multi-Post Stories</h2>
+            <ul class="canop">
+              <li>FACEBOOK</li>
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+            </p>
+          </div>
+          <div>
+            <ul class="languages-2">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/see-project.png" alt="button" class="blind">
+        </div>
+      </div>
+      <div class="card1">
+        <div class="snapshooti">
+          <img src="./images/snapshot.png" alt="snapshot" class="responsive" />
+        </div>
+        <div class="word-title">
+          <div class="w-title">
+            <h2 class="name">Facebook 360</h2>
+            <ul class="canop">
+              <li>FACEBOOK</li>
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+            </p>
+          </div>
+          <div>
+            <ul class="languages">
+              <li>html</li>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/Hover.png" alt="button" class="see">
+        </div>
+      </div>
```

Comment:
- [ ] You have used an image in place of a `<button>` tag for the "See Project" button. Please remove the image and use a button tag instead. Do this for all your images.
- [ ] Also kindly implement the button default, hover, and active states as shown in the Figma design in my earlier suggestions ☝️